src/components/ui/Button.astro

```diff
@@ -37,11 +37,11 @@ const {
     shaText = "",
     } = Astro.props;
 
-const localClassList = `${colorMap[color]} font-bold ${sizeMap[size]} text-white drop-shadow-[0_4px_1px_rgba(0,0,0,0.2)]`;
+const localClassList = `${colorMap[color]} object-contain font-bold ${sizeMap[size]} text-white drop-shadow-[0_4px_1px_rgba(0,0,0,0.2)]`;
```

Comment:
Was something breaking without `object-contain`?

Reply:
not really, this is just a remainder of debugging. I just removed it :D